code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -6,35 +8,26 @@
 	icon_state = "cutout_basic"
 	w_class = WEIGHT_CLASS_BULKY
 	resistance_flags = FLAMMABLE
-	/// Possible restyles for the cutout, add an entry in change_appearance() if you add to here
-	var/list/possible_appearances = list()
+	item_flags = NO_PIXEL_RANDOM_DROP
 	/// If the cutout is pushed over and has to be righted
 	var/pushed_over = FALSE
 	/// If the cutout actually appears as what it portray and not a discolored version
 	var/deceptive = FALSE
+	/// Should we load a cutout datum at the start?
+	var/starting_cutout
```

Comment:
If this is a bool give it a default value

Reply:
its not